sky/build/sdk_xcode_harness/Local.xcconfig.tmpl

```diff
@@ -2,9 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-//
-// This file generated. Do not modify or check into version control.
-//
+// This file is overwritten by `flutter tools`.
+
+LOCAL_ENGINE={{ local_engine }}
```

Comment:
Put it after FLUTTER_ROOT so it matches the version the tools generate? \[shrug\]